src/components/category.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Loader from './loader'
+import Error from './error'
```

Comment:
Don't name it `Error`, it's a reserved keyword. Might cause probs. So change this to something like "ErrorComponent".